src/App.tsx

```diff
@@ -15,47 +16,131 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+export function getReorederedGoods(
+  goods: string[],
+  sortType: SortType,
+  isReversed: boolean,
+) {
+  const visibleGoods = [...goods];
+
+  visibleGoods.sort((good1, good2) => {
+    switch (sortType) {
+      case SortType.ALPHABET:
+        return good1.localeCompare(good2);
+      case SortType.LENGTH:
+        return good1.length - good2.length;
+      default:
+        return 0;
```

Comment:
It will be easier to read
```suggestion
      case SortType.ALPHABET:
        return good1.localeCompare(good2);
        
      case SortType.LENGTH:
        return good1.length - good2.length;
        
      default:
        return 0;
```